src/core/src/main/java/org/apache/jmeter/gui/action/Duplicate.java

```diff
@@ -56,7 +56,8 @@ public void doAction(ActionEvent e) {
         JMeterTreeNode currentNode = treeListener.getCurrentNode();
         JMeterTreeNode parentNode = (JMeterTreeNode) currentNode.getParent();
         JMeterTreeModel treeModel = instance.getTreeModel();
-        for (int nodeIndex = copiedNodes.length - 1; nodeIndex >= 0; nodeIndex--) {
+        int copiedNodesCount = copiedNodes.length;
+        for (int nodeIndex = copiedNodesCount - 1; nodeIndex >= 0; nodeIndex--) {
```

Comment:
`copiedNodes.length` was computed one-time only, there's no point in adding a variable